packages/workers-shared/package.json

```diff
@@ -58,7 +58,7 @@
 	},
 	"devDependencies": {
 		"@cloudflare/eslint-config-worker": "workspace:*",
-		"@cloudflare/vitest-pool-workers": "^0.5.41",
+		"@cloudflare/vitest-pool-workers": "^0.7.0",
```

Comment:
why not `^0.8.19` like in all the other changes?

Reply:
I did do that but I had to downgrade it (see https://github.com/cloudflare/workers-sdk/pull/9033/commits/a706a0a4ab05e88c57b4c59120aa03321894a569) as the newer version was causing some wrangler incompatibility issues

I figured that it'd be better not to scope creep things here and just dump the dependency to the higher minor it could go